test/dlc_tests/sanity/test_efa.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+import pytest
+
+from test.test_utils import CONTAINER_TESTS_PREFIX, LOGGER, is_tf2
+from test.test_utils.ec2 import get_ec2_instance_type
+
+EFA_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p3dn.24xlarge", processor="gpu")
+
+
+@pytest.mark.model("N/A")
+@pytest.mark.integration("EFA")
+@pytest.mark.parametrize("ec2_instance_type", EFA_EC2_GPU_INSTANCE_TYPE, indirect=True)
+def test_sanity(training, ec2_connection, region, gpu_only, py3_only):
+    test_script = os.path.join(CONTAINER_TESTS_PREFIX, "test_efa_sanity.py")
+    run_efa_test(training, ec2_connection, region, test_script)
```

Comment:
Do the tests in the `test_efa_sanity.py` file require the use of an EFA-enabled EC2 instance?

If not, this test could be run with a smaller instance type, or possibly by just "docker pull"-ing this image into the same CB job, and examining its contents, if that is what the test does.

Reply:
Discussed offline. Tests in efa_sanity do need EFA enabled.